perses/rjmc/geometry.py

```diff
@@ -472,32 +483,10 @@ def _angle_logp(self, theta, angle, beta):
         beta : float
             1/kT or inverse temperature
         """
-        k_eq = angle.type.k*units.kilocalories_per_mole/(units.radians**2)
-        theta0 = angle.type.theteq*units.radians
-        sigma = beta*2.0/np.sqrt(2.0*k_eq/k_eq.unit)
-        logp = stats.distributions.norm.logpdf(theta/theta.unit, theta0/theta0.unit, sigma)
-        return logp
-
-    def _torsion_logp(self, atom, xyz, torsion, atoms_with_positions, positions, beta):
-        """
-        Utility function for calculating the unnormalized probability of a torsion angle
-        """
-        if torsion.atom1 == atom:
-            bond_atom = torsion.atom2
-            angle_atom = torsion.atom3
-            torsion_atom = torsion.atom4
-        else:
-            bond_atom = torsion.atom3
-            angle_atom = torsion.atom2
-            torsion_atom = torsion.atom1
-
-        internal_coordinates = self._autograd_ctoi(xyz, positions[bond_atom.idx], positions[angle_atom.idx], positions[torsion_atom.idx])
-        beta = beta/beta.unit
-        gamma = torsion.type.phase
-        V = torsion.type.phi_k
-        n = torsion.type.per
-        q = np.exp(-beta*(V/2.0)*(1+np.cos(n*internal_coordinates[2]-gamma)))
-        return q
+        k_eq = angle.type.k*units.kilocalories_per_mole/(units.degrees**2)
```

Comment:
Yeah. The `ParmEd` value does not have units, but is expressed in `kcal/mol/degrees**2`. So, when I pull that number out of the attribute, I immediately assign it the correct units (I'm not converting anything there). Otherwise, if I had `radians**2` there, then I'd be using the wrong unit.

It looks like in the code you quoted, the `ParmEd` value is in degrees, and has to be converted to radians to work in OpenMM. However, the explicit conversion just has to happen because `ParmEd` is not using unit-bearing quantities, no?